clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtilSpec.groovy

```diff
@@ -430,4 +439,64 @@ class GCEUtilSpec extends Specification {
       ["devstorage.read_only"]                                     | ["https://www.googleapis.com/auth/devstorage.read_only"]
       ["https://www.googleapis.com/auth/logging.write", "compute"] | ["https://www.googleapis.com/auth/logging.write", "https://www.googleapis.com/auth/compute"]
   }
+
+  void "getNextSequence should handle roll-over properly"() {
+    setup:
+      def computeMock = Mock(Compute)
+      def credentials = new GoogleCredentials(PROJECT_NAME, computeMock)
+      def regionsMock = Mock(Compute.Regions)
+      def regionsGetMock = Mock(Compute.Regions.Get)
+      def regionReal = new Region(zones: [ZONE_URL_1, ZONE_URL_2])
+      def instanceGroupManagersMock = Mock(Compute.InstanceGroupManagers)
+      def instanceGroupManagersListZone1Mock = Mock(Compute.InstanceGroupManagers.List)
+      def instanceGroupManagersListZone2Mock = Mock(Compute.InstanceGroupManagers.List)
+      def instanceGroupManagerReal997 = new InstanceGroupManager(name: "$CLUSTER_NAME-v997", creationTimestamp: "2015-12-24T12:06:46.103-08:00")
```

Comment:
For simplicity, can you make the creation timestamps something very vanilla, like "2000-01-01T00:00:00.997-00:00" where 997 corresponds to its respective cluster version